features/feature-main/src/main/java/com/ownsnoretracker/feature_main/presentation/MainHomeScreen.kt

```diff
@@ -16,7 +34,44 @@ fun MainHomeScreen(
     Column(
         Modifier
             .fillMaxSize()
-            .background(mainTheme)) {
-        Text(text = "main")
+            .background(mainTheme)
+    ) {
+        var triggerAnimation by remember { mutableStateOf(false) }
+
+        Box(
+            contentAlignment = Alignment.TopCenter,
+            modifier = Modifier
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line